build/fbcode_builder/patches/zlib_dont_build_more_than_needed.patch

```diff
@@ -1,16 +1,16 @@
 --- a/CMakeLists.txt
 +++ b/CMakeLists.txt
-@@ -183,8 +183,7 @@
+@@ -147,8 +147,7 @@
      set(ZLIB_DLL_SRCS ${CMAKE_CURRENT_BINARY_DIR}/zlib1rc.obj)
  endif(MINGW)
 
--add_library(zlib SHARED ${ZLIB_SRCS} ${ZLIB_ASMS} ${ZLIB_DLL_SRCS} ${ZLIB_PUBLIC_HDRS} ${ZLIB_PRIVATE_HDRS})
--add_library(zlibstatic STATIC ${ZLIB_SRCS} ${ZLIB_ASMS} ${ZLIB_PUBLIC_HDRS} ${ZLIB_PRIVATE_HDRS})
+-add_library(zlib SHARED ${ZLIB_SRCS} ${ZLIB_DLL_SRCS} ${ZLIB_PUBLIC_HDRS} ${ZLIB_PRIVATE_HDRS})
```

Comment:
Just have one question, how were this changes generated, or why ZLIB_ASMS was removed?

Reply:
I did not remove ZLIB_ASMS. This is a format for “patch’s patch”, not a simple patch, that may cause confusion . I just move old patch to new version(1.2.13) of zlib.You can check through path“ build/fbcode_builder/patches/zlib_dont_build_more_than_needed.patch”. 
BTW, I use “git diff” to generate this patch.

Reply:
Although building failed ,this patch should be merged. Also, with bug “403 error for downloading libmnl” fixed, build problems can be resolved.